docs/ccampbell-intuit.md

```diff
@@ -0,0 +1,8 @@
+# Chris Campbell
+
+```swift
+let name = "Chris Campbell"
+let occupation = "Staff Software Engineer"
+let location = "Cocoa, Florida"
+let ssn = "XXX-XX-XXXX"
```

Comment:
You should use numbers, such as 999-99-9999 to indicate that only numbers are allowed in SSN.